src/HttpTracer.TestApp/ViewModels/MainPageViewModel.cs

```diff
@@ -1,84 +1,140 @@
-﻿using Prism.Commands;
-using Prism.Mvvm;
-using Prism.Navigation;
+﻿using Prism.Navigation;
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.Linq;
+using System.Net;
 using System.Net.Http;
-using System.Text;
-using System.Threading;
 using System.Threading.Tasks;
 using System.Windows.Input;
 using HttpTracer.Logger;
+using MvvmHelpers;
+using Newtonsoft.Json.Linq;
 using Xamarin.Forms;
 
 namespace HttpTracer.TestApp.ViewModels
 {
     public class MainPageViewModel : ViewModelBase
     {
+        private const string Domain = "reqres.in";
         private const HttpMessageParts DefaultHttpTracerVerbosity =
             HttpMessageParts.RequestAll | HttpMessageParts.ResponseHeaders;
+
+        private ICommand _logToScreenCommand;
+        public ICommand LogToScreenCommand => _logToScreenCommand =
```

Comment:
I discovered these are actually unnecessary. The bound button inits on construction, so just new it up in the constructor.